### DISCUSSION TITLE
Hepatic impairment population

### DISCUSSION THREAD

**Discussion by Nathalie06** [Open-Systems-Pharmacology/Forum#836] [Questions & Problems] [answered]
Dear all, 

I would like to perform simulations in hepatic impairment population (Child-Pugh class A, B and C).
Is someone already develop and share these populations ? I have not found.
Othewise, is it possible to change the portal vein blood flow in PK-Sim ? because arterial hepatic blood flow increases in these populations but portal vein blood flow decreases...

Thanks in advance,
Best,
Nathalie

**Comment by prvmalik** [comment, marked as answer]
Hi Nathalie

This paper by Edginton and Willmann (2008) gives a parameterization for alterations in blood flows, GFR, PPB and so on.
doi: 10.2165/00003088-200847110-00005

This paper by Prasad et al. (2018) gives the abundance of CYP and UGT enzymes in liver cirrhosis for modeling.
doi: 10.1124/dmd.118.080523

Take care
Paul

**Comment by Nathalie06** [comment]
Thanks Paul for these papers, I have read the first but not the second.

Best,
Nathalie
